lxd/provider.go

```diff
@@ -125,37 +124,6 @@ func Provider() terraform.ResourceProvider {
 				},
 			},
 
-			"address": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Removed:  "Use `lxd_remote.address` instead.",
-			},
-
-			"scheme": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Removed:  "Use `lxd_remote.scheme` instead.",
-			},
-
-			"port": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Removed:  "Use `lxd_remote.port` instead.",
-			},
-
-			"remote": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Removed:  "Use `lxd_remote.name` instead.",
-			},
-
-			"remote_password": {
-				Type:      schema.TypeString,
-				Sensitive: true,
-				Optional:  true,
-				Removed:   "Use `lxd_remote.password` instead.",
-			},
-
```

Comment:
Why are these being removed? If everything is still functional with them included, we should keep them and defer removal for a major update increment.